ai-builder/labs/README.md

```diff
@@ -27,16 +27,12 @@ For prediction, text classification, and object detection scenarios you will nee
 
 #### Step 1 : Import AIBuilderLabSolution_1_0_0_1 solution to the Dataverse environment
 
-This creates 3 Dataverse entities - Object Detection Product, Health Feedback, and Online Shopping Intent.
-
-- Object Detection Product
-- Health Feedback
-- Online Shopping Intent
-- Travel feedback
+This will create 5 Dataverse tables - Object Detection Product , Health Feedback, Online Shopping Intent, Travel feedback, Expenses
 
 #### Step 2 : Upload data to the entities created in step 1
 
 - **Prediction** : Follow the instructions in [Prediction data preparation](https://docs.microsoft.com/ai-builder/binary-classification-data-prep) to upload the **Online Shopping Intent** data.
-- **Category classification** : Go to **Lab Data/Text Classification** folder within the lab files and then upload data from **pai_healthcare_feedbacks**. Follow the instructions in [Before you build a category classification model](https://docs.microsoft.com/ai-builder/before-you-build-text-classification-model).
-- **Entity extraction** : Go to **Lab Data/EntityClassification** folder with the lab files and then upload data from **aib_travelfeedback**.  Follow the instructions in [Use sample data to do entity extraction](https://docs.microsoft.com/ai-builder/entity-extraction-sample-data).
-- **Object detection** : Go to **Lab Data/ObjectDetection** folder and then upload data from **aib_objectdetectionproducts**. Follow the same instructions as above for data upload.
+- **Category classification** : Go to **Lab Data/CategoryClassification** folder within the lab files and then upload data from **pai_healthcare_feedbacks**. Follow the instructions in [Before you build a category classification model](https://docs.microsoft.com/ai-builder/before-you-build-text-classification-model).
+- **Entity extraction** : Go to **Lab Data/EntityExtraction** folder with the lab files and then upload data from **aib_travelfeedback**.  Follow the instructions in [Use sample data to do entity extraction](https://docs.microsoft.com/ai-builder/entity-extraction-sample-data).
```

Comment:
replace link with (/ai-builder/entity-extraction-sample-data).

Reply:
Hi Angie - are you sure this will work? This is a readme only visible on GitHub. It does not appear on https://docs.microsoft.com/

Reply:
I'll probably need to roll them back. Waiting for confirmation. Also, line 35 says **pai_healthcare_feedbacks**. Is feedbacks really plural? In correct grammar, the plural of feedback is feedback.